sites/main-site/src/content/blog/2024/seqera-containers-part-2.mdx

```diff
@@ -0,0 +1,646 @@
+---
+title: 'Migration from Biocontainers to Seqera Containers: Part 2'
+subtitle: "nf-core containers automation: how it'll all work behind the curtain"
+pubDate: 2024-10-02T09:20:00+01:00
+headerImage: https://images.unsplash.com/photo-1711109631679-c9e1094d3118
+headerImageAlt: Photo by Rafael Garcin on Unsplash
+authors:
+    - 'ewels'
+    - 'edmundmiller'
+label:
+    - 'modules'
+    - 'wave'
+    - 'seqera containers'
+embedHeaderImage: false
+---
+
+import creation_flow from '@assets/images/blog/seqera-containers-part-2/creation_flow.excalidraw.svg?raw';
+import renovate_flow from '@assets/images/blog/seqera-containers-part-2/renovate_flow.excalidraw.svg?raw';
+
+import { Image } from 'astro:assets';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+# Introduction
+
+[Wave](https://seqera.io/wave/) is an open-source software that Seqera released at the [Nextflow Summit Barcelona](https://www.youtube.com/watch?v=PTbiCVq0-sE) in the fall of 2022.
+It enables users to create software containers on demand, to rapidly speed up the development cycle.
+Wave has a number of tricks up its sleeve, but in its simplest form, it can be passed a list of conda packages
+and build a container with those packages installed.
+
+[Seqera Containers](https://seqera.io/containers/) is a service for the bioinformatics community that's built on top of Wave,
+providing an easy user interface to quickly create a container with a set of conda packages
+and share it with others.
+
+In nf-core, we've been excited to adopt Wave more fully in our workflows and have been looking for the right way to do it.
+With the announcement of Seqera Containers we felt it was the right time to put in the effort to migrate
+our containers to be built using Wave, using Seqera Containers to host the container images for our modules.
+
+You can read more about our motivation for this change in [Part 1 of this blog post](https://nf-co.re/blog/2024/seqera-containers-part-1).
+Here, in Part 2, we will dig into the technical details: how it all works behind the curtain.
+You don't need to know or understand any of this as an end-user of nf-core pipelines,
+or even as a contributor to nf-core modules, but we thought it would be interesting to share the details.
+It's mostly to serve as an architectural plan for the nf-core maintainers and infrastructure teams.
+
+# The end goal
+
+Before we dig into how the details of how the automation will work, let's summarise the end goal of this migration:
+
+> "What we call the beginning is often the end. And to make an end is to make a beginning. The end is where to start from."<br /><small>_T.S. Eliot_</small>
+
```

Comment:
I don't think this is useful,  and just takes up space for no purpose

Reply:
Your review was too slow, I already removed it 😆